### PR TITLE
Qweb: honor translated account names

### PR DESCRIPTION
**Pull request by ghost**
The skeleton of the report is available in French and `account.account.name` is a translatable field, so it should be translated as well. We've had issues due to that at Open Net Sàrl before. I based my SQL additions on [the request in ir.translation itself](https://github.com/odoo/odoo/blob/216d328a11f254a3c037eb9e23ab362b984ca0ba/openerp/addons/base/ir/ir_translation.py#L403-L432), and it can fall back on displaying the untranslated value.

NOTE: I haven't added my name and my company's to the README because it would likely conflict with #224 later. In the event that my other pull request was rejected, I would like [lbeopennet:9.0-translated-copyright](https://github.com/open-net-sarl/account-financial-reporting/tree/9.0-translated-copyright) to be integrated in order for the copyright notice to be updated. In any other case, please disregard.

### CODE REVIEW

File: account_financial_report_qweb/report/general_ledger.py [hunk 1]
```diff
@@ -492,6 +497,7 @@ def _inject_account_values(self):
         query_inject_account_params += (
             self.id,
             self.env.uid,
+            self.env.lang,
```
Comment (pedrobaeza) [inline account_financial_report_qweb/report/general_ledger.py:500]: Shouldn't this be `self.env.context.get('lang', self.env.lang)`?

**Comment by pedrobaeza** [comment]
About the copyright, your change is very little (a bug with its patch) to be considered as a co-author/contributor. If you make more changes or contributions, then it can be considered.

**Comment by pedrobaeza** [comment]
Anyways, your name will appear and counts as contribution in git commit history and in the statistics we are preparing at OCA, so every little thing counts!

File: account_financial_report_qweb/report/general_ledger.py (hunk 1 above)
Comment (ghost) [reply account_financial_report_qweb/report/general_ledger.py:500]: That would be redundant given that [`self.context.lang`](https://github.com/odoo/odoo/blob/9.0/openerp/api.py#L802) is itself nothing more than an alias for `self.context.get('lang')`.

Comment (pedrobaeza) [reply account_financial_report_qweb/report/general_ledger.py:500]: Cool! I didn't know about that "alias"

**Review by pedrobaeza — APPROVED** [review]

**Comment by ghost** [comment]
Yes you're right, I'm fine with that. Thank you very much!

**Comment by dcoopennet** [comment]
LGTM

**Comment by damdam-s** [comment]
:+1: thanks

**Comment by Garamotte** [comment]
LGTM :+1:

**Comment by jcoux** [comment]
Thanks you !
Bug fix is ok for the General Ledger Report.
But if you can do it also on the Open Items Report (similar SQL request), the bug will be fixed on all reports.
:+1: with that

**Comment by ghost** [comment]
hey thanks @jcoux, didn't think of that. I'll do it today.

**Comment by ghost** [comment]
Done.

**Comment by ghost** [comment]
(while I'm at it)

**Comment by atchuthan** [comment]
code review :+1:

**Pull request merged**